packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2561,8 +2561,8 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     if (_primaryScrollController != null && _primaryScrollController.hasClients) {
       _primaryScrollController.animateTo(
         0.0,
-        duration: const Duration(milliseconds: 300),
-        curve: Curves.linear, // TODO(ianh): Use a more appropriate curve.
+        duration: const Duration(milliseconds: 1000),
```

Comment:
In native iOS, I found duration is 700ms to 1100ms. But, it's 700ms when content's is very short (a little bigger than device's height).
In almost case, it's 850ms-1100ms. I picked 1000ms but 900ms would be nice too